### ISSUE TITLE
converts plus zone to minus zone

### ISSUE THREAD

**Issue by prinze77** [moment/luxon#711]
const { DateTime } = require('luxon');

console.log(DateTime.local().zoneName);
console.log(DateTime.local().setZone('ETC/GMT+5').zoneName);

Output:
Asia/Nicosia
UTC-5

Expected:
Asia/Nicosia
UTC+5

I cant understand why it converts plus 5  to minus5

**Comment by icambron** [comment]
Yeah, this is a weird old spec thing. See https://github.com/moment/luxon/issues/302. They just defined the IANA zone to be backward, so GTM+5 = UTC-5. 

It's busted conceptually, but it's what it's supposed to do, so we can't change it. Annoying, Chrome doesn't accept `Etc/UTC+5`, though we should probably support that directly in Luxon itself.

(For future me: looks like we can actually remove the explicit GMT parsing code, since Chrome now supports `ETC/GMT+x`, and just replace it with explicit _UTC_ parsing code, which is the same but with the sign flipped)
